Dockerfile

```diff
@@ -49,6 +49,8 @@ ENV OPENVPN_USERNAME=**None** \
     GLOBAL_APPLY_PERMISSIONS=true \
     TRANSMISSION_HOME=/config/transmission-home \
     TRANSMISSION_RPC_PORT=9091 \
+    TRANSMISSION_RPC_USERNAME= \
```

Comment:
Why this sudden addition here?

Reply:
A best practice in bash is to add `set -e -u -o pipefail` in order to have a stricter usage of bash.
some exemple: https://gist.github.com/atomlab/65e70d6f9d0ed5b3aeb3fc76ff28255a
some information about strict mode: http://redsymbol.net/articles/unofficial-bash-strict-mode/

these two lines are added to avoid an unbound variable error due to `set -u` and a non existing variable.
the `set -u -e -o pipefail` will ease debug and prevent subtle bugs. 
Dockerfile will initialise in container environment, the variable to empty or default value.
the results of  `[ -z ${TRANSMISSION_RPC_USERNAME} ]  and `[ -n ${TRANSMISSION_RPC_USERNAME} ]` will remain the same 

the var is dumped in [openvpn start's script](https://github.com/edgd1er/docker-transmission-openvpn/blob/c43dd6ab2304662f4e000010e7e96303e5f5307f/openvpn/start.sh#L174), defaulting it to a known value is a good practice. If a set -u was added in that script, execution would failed. 

https://github.com/haugene/docker-transmission-openvpn/blob/c43dd6ab2304662f4e000010e7e96303e5f5307f/openvpn/start.sh#L174

this change is not mandatory for the feature to be available. I could commit a revert for this two lines. just let me know what do you prefer.